.travis.yml

```diff
@@ -15,3 +15,7 @@ env:
     - SITE_PAGE_API_URL=https://gds-cyber-security.cloudapps.digital/api/pages.json
 
 script: ./travis.sh
+
+branches:
+  only:
+  - master
```

Comment:
Travis makes Daniel post to:
- #bot-testing for PRs and merges, using the PR build
- configured owners, using the build from the master branch

Does your change mean it will stop posting to #bot-testing? I think it's useful to have Daniel post to #bot-testing as a sanity check when adding a new set of docs 😁

Reply:
Yeah this probably doesn't make sense :sweat_smile: I'll remove.